tools/validators/instance_validator/universe_generation/ontology_validation.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Uses ontology universe to validate parsed instance data."""
+
+from __future__ import print_function
+from yamlformat.validator import findings_lib
+
+class EntityInstance(findings_lib.Findings):
+  """Uses information from the generated ontology universe to validate
+  an entity.
```

Comment:
entity --> entity instance.

Add the following to the doc:
`An entity instance is composed of a an id and a type.
For example: 
{'id': 'FACILITIES/123456', 'type': 'FACILITIES/BUILDING'}.
`

Also you can validate that when en entity_instance is passed, that the id and the type are present.

+ Add unit tests for the id validation/presence

Reply:
Done. The presence of the ID is caught by the syntax portion, when the YAML file is first being parsed by the strictyaml schema.